src/transformers/clip.rs

```diff
@@ -240,37 +234,26 @@ impl<B: Backend> ClipAttention<B> {
     fn shape(&self, xs: Tensor<B, 3>, seq_len: usize, bsz: usize) -> Tensor<B, 4> {
         xs.reshape([bsz, seq_len, self.num_attention_heads, self.head_dim])
             .swap_dims(1, 2)
-        // .contiguous() // TODO: Figure out if this is needed or if we can abstract over memory
```

Comment:
In Burn we don't have to worry about contiguous.

<img width="711" alt="Screenshot 2023-11-22 at 01 08 39" src="https://github.com/OxideAI/diffusers-burn/assets/2475932/91ba0f84-49ac-4971-aabf-90909debde99">